source/interprocedural_analyses/taint/modelParser.ml

```diff
@@ -23,6 +23,60 @@ open Statement
 open Domains
 open ModelParseResult
 
+module PythonVersion = struct
+  (* Not putting the functions there to prevent circular dependency errors *)
+  include Configuration.PythonVersion
+
+  let parse_from_tuple ~tuple =
```

Comment:
nitpick: since we only take a single parameter and it's pretty obvious from the name, I wouldn't use a named parameter
```suggestion
  let parse_from_tuple tuple =
```